src/routes/news/components/__tests__/LocalNewsList.spec.js

```diff
@@ -25,7 +25,7 @@ describe('LocalNewsList', () => {
     t={t}
     language={language}
   />
-  const date = moment.tz('2020-03-20 17:50:00', 'GMT')
+  const date = moment('2017-11-18T09:30:00.000Z')
```

Comment:
Do that in the [other PR](https://github.com/Integreat/integreat-webapp/pull/370)

Reply:
Whoops, yeah, that was the plan -.-'

Reply:
But I guess, I won't take the effort now, to swap this change to the other PR tbh :see_no_evil: